blackbox/docs/protocols/http.txt

```diff
@@ -16,9 +16,12 @@ whereby the statement is sent as value associated to the key ``stmt``.
 
 A simple `SELECT` statement can be submitted like this::
 
-    sh$ curl -sSXPOST '127.0.0.1:4200/_sql?pretty' -d '{
-    ... "stmt":"select name, position from locations order by id limit 2"
-    ... }'
+    sh$ curl -H 'Content-Type: application/json' \
```

Comment:
since the code block are executed using python doctests, the `...` are required for multi-line command, otherwise the second line would already be interpreted as result:

you can run the doctests with  `./gradlew itest`

this results in:

```
----------------------------------------------------------------------
File "/Users/christian/sandbox/crate/crate/blackbox/docs/src/crate/../../protocols/http.txt", line 19, in http.txt
Failed example:
    import subprocess;print(subprocess.check_output(r"""curl -H 'Content-Type: application/json' \
    """,stderr=subprocess.STDOUT,shell=True).decode("utf-8"))
Exception raised:
    Traceback (most recent call last):
      File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/doctest.py", line 1318, in __run
        compileflags, 1), test.globs)
      File "<doctest http.txt[0]>", line 2, in <module>
        """,stderr=subprocess.STDOUT,shell=True).decode("utf-8"))
      File "/opt/local/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/subprocess.py", line 617, in check_output
        raise CalledProcessError(retcode, process.args, output=output)
    subprocess.CalledProcessError: Command 'curl -H 'Content-Type: application/json' \
    ' returned non-zero exit status 2
```